pkg/kubelet/container/runtime.go

```diff
@@ -97,6 +97,8 @@ type Runtime interface {
 	RemoveImage(image ImageSpec) error
 	// Returns Image statistics.
 	ImageStats() (*ImageStats, error)
+	// Cadvisor FS Label
```

Comment:
godoc style (beginning with `ImageLabel`) and a little more information would be nice.